packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -779,6 +779,7 @@ export class Resolver {
       case 'image':
         return {
           component: 'image',
+          clearable: true,
```

Comment:
Ah, simple enough. 👍

Reply:
@Enigmatical this did present some style issues that might have to be fixed in  https://github.com/tinacms/tinacms/pull/2261